src/properties/competitionbot/intake.properties

```diff
@@ -2,3 +2,5 @@ dream.intake.sideBelt.forward = 1
 dream.intake.sideBelt.reverse = -1
 dream.intake.mainBelt.forward = 1
 dream.intake.mainBelt.reverse = -1
+rollerbar.intake.in = -1
+rollerbar.intake.out = 1
```

Comment:
I would suggest naming these property `rollerbar.intakeSpeed.in` and `rollerbar.intakeSpeed.out`. Just calling it "intake" doesn't really communicate what the property means.

Don't forget to change this in `IntakeConfig` as well.